src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -748,6 +748,45 @@ def start_agents(self, agent_groups=None):
                     info["access_points"])
             self.start_agent_managers()
 
+    def stop_group_servers(self, server_group, hosts):
```

Comment:
The main reason I did it this way was to avoid exposing the get_server_manager() stuff to the test code, which only knows about the server group. Are there other examples of tests interacting directly with manager groups?

Reply:
The test code has access to the server information through the `self.server_managers` list of `DaosServerManager` objects.  This value can be obtained through each `DaosServerManager` object's server configuration via `self.server_managers[0].get_config_value("name")`.  In terms of examples a search for "self.server_managers[0]" or "self.server_managers[0].get_config_value" in the ftest files should give you some examples.